### PR TITLE
Draft post of 2020-02-07-status-update

### PR DESCRIPTION
**Pull request by cmccrisken-puppet**
Appreciate some feedback on the first draft for the status update blog to go out on 2020-02-07.

Can we enable anchors in other blog posts? It would be nice [here](https://github.com/puppetlabs/iac/compare/2020-02-07_blog_post?expand=1#diff-afe49f640447e6a672de6f9921b395c1R22) if I could link on **Content Triage** to my previous blog post.

I also created a (_very rough_) script to take the output of Kerminator's `! modules released` command and convert it to markdown with links to The Forge:
```sh
#!/bin/bash

FORGE_BASE_URI="https://forge.puppet.com"
CHANGELOG_URI="changelog#"

print_usage() {
	echo -e "\nUSAGE:\n"
	echo "Run kerminator in Slack to get the last 14 days of released modules: !modules released 14."
	echo "Save the output of kerminator to a file."
	echo "Run this script with the first arg pointing to the file above."
	echo -e "\n./kerminator_to_blog_md.sh /path/to/kerminator_output.txt\n"
}

if [ -z "$1" ] || [ `test -f "$1"` ]; then
	print_usage
	exit 1
fi

OUTPUT=""

while read line; do
	moduleName=$(echo -ne $line | cut -d ' ' -f 1 | sed 's/-/\//')
	version=$(echo -ne "v"; echo -ne $line | cut -d ' ' -f 3)
	OUTPUT="$OUTPUT\n- [$moduleName]($FORGE_BASE_URI/$moduleName) ($version)"
done < "$1"

echo -e $OUTPUT | sort
```
I was trying to link to specific sections in the CHANGELOG with the date & version, but unfortunately there's not enough consistency yet in the format, most of them were ending up as dead links. Would've been nice...

Not sure where this script should live, if someone could recommend somewhere?

### CODE REVIEW

**Comment by cmccrisken-puppet** [comment]
Script I/O:
```
# /tmp/modules.txt
puppetlabs-websphere_application_server / 2.0.1 / 2020-02-06T07:42:46.000Z / [supported]
puppetlabs-bootstrap / 2.1.0 / 2020-02-04T17:32:33.000Z
puppetlabs-java / 6.1.0 / 2020-02-03T21:36:59.000Z / [supported]
puppetlabs-reboot / 2.4.0 / 2020-02-03T20:28:46.000Z / [supported]
puppetlabs-accounts / 6.1.0 / 2020-02-03T15:18:50.000Z / [supported]
puppetlabs-puppet_agent / 3.0.1 / 2020-01-29T18:54:52.000Z / [supported]
puppetlabs-facts / 1.0.0 / 2020-01-28T17:16:36.000Z
puppetlabs-kubernetes / 5.1.0 / 2020-01-28T15:11:13.000Z / [supported]
puppetlabs-cd4pe / 1.4.1 / 2020-01-27T21:32:08.000Z
puppetlabs-puppet_agent / 3.0.0 / 2020-01-27T17:20:39.000Z / [supported]
puppetlabs-mount_iso / 3.0.0 / 2020-01-23T10:15:34.000Z
puppetlabs-apache / 5.4.0 / 2020-01-23T08:57:46.000Z / [supported]
puppetlabs-concat / 6.2.0 / 2020-01-21T16:08:00.000Z / [supported]
puppetlabs-exec / 0.7.0 / 2020-01-21T12:36:20.000Z
```
```
./kerminator_to_blog_md.sh /tmp/modules.txt

- [puppetlabs/accounts](https://forge.puppet.com/puppetlabs/accounts) (v6.1.0)
- [puppetlabs/apache](https://forge.puppet.com/puppetlabs/apache) (v5.4.0)
- [puppetlabs/bootstrap](https://forge.puppet.com/puppetlabs/bootstrap) (v2.1.0)
- [puppetlabs/cd4pe](https://forge.puppet.com/puppetlabs/cd4pe) (v1.4.1)
- [puppetlabs/concat](https://forge.puppet.com/puppetlabs/concat) (v6.2.0)
- [puppetlabs/exec](https://forge.puppet.com/puppetlabs/exec) (v0.7.0)
- [puppetlabs/facts](https://forge.puppet.com/puppetlabs/facts) (v1.0.0)
- [puppetlabs/java](https://forge.puppet.com/puppetlabs/java) (v6.1.0)
- [puppetlabs/kubernetes](https://forge.puppet.com/puppetlabs/kubernetes) (v5.1.0)
- [puppetlabs/mount_iso](https://forge.puppet.com/puppetlabs/mount_iso) (v3.0.0)
- [puppetlabs/puppet_agent](https://forge.puppet.com/puppetlabs/puppet_agent) (v3.0.0)
- [puppetlabs/puppet_agent](https://forge.puppet.com/puppetlabs/puppet_agent) (v3.0.1)
- [puppetlabs/reboot](https://forge.puppet.com/puppetlabs/reboot) (v2.4.0)
- [puppetlabs/websphere_application_server](https://forge.puppet.com/puppetlabs/websphere_application_server) (v2.0.1)
```

File: _drafts/status-update.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Kubernetes v5.1.0
+The [CNCF](https://www.cncf.io/) have certified the latest version (v5.1.0) of the [puppetlabs/kubernetes](https://forge.puppet.com/puppetlabs/kubernetes) module with [Kubernetes](https://kubernetes.io/) v1.16.6.
+Check out the [module on The Forge](https://forge.puppet.com/puppetlabs/kubernetes) to see how to bootstrap K8 clusters and save yourself time and effort.
+You may also be interested in Kubetool (bundled with the module and as a [Docker image](https://hub.docker.com/r/puppet/kubetool)) which auto-generates the Hiera security parameters, the discovery token hash, and other configurations for your Kubernetes cluster.
+
+## There's a buzz about Honeycomb.io
+We're very excited to be in the POC stage rolling out [Honeycomb](https://www.honeycomb.io/) to collect data from our CI / Test infrastructure.
+This is going to give us much more visibility in to a number of important metrics and allow us to make the test process for modules a lot slicker.
+We were very excited to be working with this very cool solution and we're already starting to get some interesting insights.
+A more in depth blog post about this in the near future - don't fly too far away!
```
Comment (DavidS) [inline _drafts/status-update.md]: Add this screenshot as a teaser - "travis and appveyor runs for a pdksync run"
2020-02-07_09MS+0000_590x447

File: _drafts/status-update.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Module Statistics Dashboards
+A vital part of the Content Triage Process is visibility in to the status of the modules:
+- How many PRs are open?
+- What is the status of the last `N` test results?
+- When was the last comment from a contributor?
+
+There are a lot of modules on [The Forge](https://forge.puppet.com/), so we need a way to quickly view all this data.
```
Comment (DavidS) [inline _drafts/status-update.md]: link to https://forge.puppet.com/supported to make sure we're not claiming responsibility for *everything* :-D

File: _drafts/status-update.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "7th Feb 2020: IAC Team Status Update"
+author: criskoduck
+categories:
+  - team
+  - status
```
Comment (DavidS) [inline _drafts/status-update.md]: :+1:

File: _drafts/status-update.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  - kubernetes
+  - tooling
+  - personal
+  - github
+  - dashboards
+  - honeycomb
+  - metrics
+---
+
+The IAC team have been working on a lot of interesting things these past two weeks. Here's a brief overview of what we've been up to:
```
Comment (DavidS) [inline _drafts/status-update.md]: Mention the obvious addition of the blog that is currently being read :-D

File: _drafts/status-update.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+## Module Statistics Dashboards
+A vital part of the Content Triage Process is visibility in to the status of the modules:
+- How many PRs are open?
+- What is the status of the last `N` test results?
+- When was the last comment from a contributor?
+
+There are a lot of modules on [The Forge](https://forge.puppet.com/), so we need a way to quickly view all this data.
+We have made some great improvements to our [Community Management](https://puppetlabs.github.io/community_management/) dashboards recently:
+- More accurate reporting on module metrics
+- Integration with [Github Actions](https://github.com/features/actions) reports / artifacts
```
Comment (DavidS) [inline _drafts/status-update.md]: add the "why?": these are our release checks, and we're planning to add nightly runs, too.

File: _drafts/status-update.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (DavidS) [inline _drafts/status-update.md]: Put up the PR directly as _posts/2020-02-07-weekly-update.md - no need to back-and-forth; merge=release!

**Comment by cmccrisken-puppet** [comment]
Thanks for the feedback @DavidS - I've addressed your comment. I also enhanced the script to filter out unsupported modules and [gave it a home here](https://github.com/cmccrisken-puppet/puppet-iac-scripts/blob/master/blog/kerminator_to_blog_md.sh)

**Pull request merged**